### ISSUE TITLE
Introduce color policy

### ISSUE THREAD

**Issue by annajeannette** [NetherlandsForensicInstitute/lir-deprecated#99]
Decide upon colors for H1 and H2 data, and apply this in every plot. Make sure these correspond to colors used in publications involving LiR.

Optionally: introduce legends for these plots.

**Comment by VincentWervingsdagen** [comment]
Agree, here are some colour palletes for colour blind people: https://gist.github.com/thriveth/8560036

Blue: #377eb8
orange: #ff7f00

**Comment by PimMeulensteen** [comment]
Will be fixed in the lir 1.0 repo (see [issue 15](https://github.com/NetherlandsForensicInstitute/lir/issues/15))
